sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/tests/QueueListenerTests.cs

```diff
@@ -127,7 +129,7 @@ public async Task GetMetrics_ReturnsExpectedResult()
             QueueListener listener = new QueueListener(Fixture.Queue, null, mockTriggerExecutor.Object, new WebJobsExceptionHandler(null),
                 _loggerFactory, null, queuesOptions, queueProcessor, new FunctionDescriptor { Id = "TestFunction" }, mockConcurrencyManager.Object);
 
-            var metrics = await listener.GetMetricsAsync();
+            var metrics = await _scaleMonitor.GetMetricsAsync();
```

Comment:
Lets move the to new test class, see example:
https://github.com/Azure/azure-sdk-for-net/pull/31367/files#diff-6ef24f0d3702cc99ff467e56538a21bdcf9e22119e84e441e10417eb66237e55

Reply:
done